src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/EntityCreationRevisionNote.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
+ * Copyright 2024 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Domain\Entity\Entity;
```

Comment:
This should not be in the Entity namespace as it isn't an Entity (no id)

Reply:
It is part of an entity (ManageEntity) and hence I stored this object in this namespace. There is a point when namespaces based around programming concepts stop being usefull, and a more domain name based approach would be more descriptive..